awsconfigrules/Makefile

```diff
@@ -1,7 +1,15 @@
+ifdef ACCOUNT
+else
+        ACCOUNT := 442294194136
```

Comment:
Shouldnt this no more be hardcoded? like use env variables

Reply:
the value is coming from env variable, this is default value, if env variable is missing.